iBudget/ibudget/views.py

```diff
@@ -19,11 +19,11 @@ def get(self, request):
         """
         if request.GET['tab'] == 'fund':
             urls = AwsService.get_default_list_icons('standard_fund/')[1:]
-        if request.GET['tab'] == 'income':
+        elif request.GET['tab'] == 'income':
```

Comment:
In this specific case, it is better to use a local variable for `request.GET['tab']` and use it instead of `request.GET['tab']`

Reply:
thanks, I will change that!